boards/nrf52840dongle/include/periph_conf.h

```diff
@@ -67,11 +68,53 @@ static const uart_conf_t uart_config[] = {
  * @{
  */
 static const pwm_conf_t pwm_config[] = {
-    { NRF_PWM0, { GPIO_PIN(0, 6), GPIO_PIN(0, 8), GPIO_PIN(1, 9), GPIO_PIN(0, 12) } }
+    {
+        NRF_PWM0,
+        {
+            GPIO_PIN(0, 6),
+            GPIO_PIN(0, 8),
+            GPIO_PIN(1, 9),
+            GPIO_PIN(0, 12),
+        },
+    },
 };
 #define PWM_NUMOF           ARRAY_SIZE(pwm_config)
 /** @} */
 
+/**
+ * @name    SPI configuration
+ * @{
+ */
+static const spi_conf_t spi_config[] = {
+    {
+        .dev  = NRF_SPIM0,
+        .sclk = GPIO_PIN(0, 20),
+        .mosi = GPIO_PIN(0, 22),
+        .miso = GPIO_PIN(0, 24),
+#ifdef ERRATA_SPI_SINGLE_BYTE_WORKAROUND
+        .ppi = 0,
+#endif
```

Comment:
This is only needed for nrf52832

Reply:
of course, overlooked this while copying the default config... thx